#goal 
oea_msgs/Oea_path plan_goal # carries path and zone tolerance information
---
#result
bool result_state
string error_string
---
#feedback
uint8 hw_state
